### PR TITLE
simplifies preview's UI and adds browser command

### PR DESCRIPTION
**Pull request by kzidane**

### CODE REVIEW

**Comment by kzidane** [comment]
browser

cc @dmalan

**Comment by kzidane** [comment]
Unfortunately, no easy way to hide the "Open in Preview" item as C9's link_handler [doesn't expose](https://github.com/c9/core/blob/master/plugins/c9.ide.terminal/link_handler.js#L284) the containing [menu](https://github.com/c9/core/blob/master/plugins/c9.ide.terminal/link_handler.js#L123).

Shouldn't be a problem though as, once our previewer is disabled online, clicking a URL in a terminal and choosing "Open in Preview" should open the URL in a built-in browser tab (instead of trying to start apache, etc.). 

Would that work for us?

**Comment by kzidane** [comment]
@dmalan

File: simple.js [hunk 1]
```diff
@@ -705,6 +707,44 @@ define(function(require, exports, module) {
...
+                    // ensure URL is given
+                    if (!_.isArray(args) || args.length !== 2 || !_.isString(args[1]))
+                        return console.log("Usage: c9 exec browser URL");
+
+                    // open URL in built-in browser tab
+                    preview.openPreview(args[1], null, true);
+                }
+            }, plugin);
+
+            // TODO determine if possible to do for "c9 exec browser" command only
```
Comment (dmalan) [inline simple.js]: What do you mean by this TODO exactly?

**Comment by dmalan** [comment]
> clicking a URL in a terminal and choosing "Open in Preview" should open the URL in a built-in browser tab (instead of trying to start apache, etc.)

Sounds reasonable!

**Comment by dmalan** [comment]
:shipit:

File: simple.js (hunk 1 above)
Comment (kzidane) [inline simple.js]: Was hoping to find a way to simplify only when the browser is opened through the command (as with phpliteadmin). The preview plugin also supports other previewers (e.g., markdown). Would be nice not to apply this simplification to them too.

**Comment by dmalan** [comment]
Is this ready to be merged in, @kzidane?

**Comment by kzidane** [comment]
I think it was, but change of plans (we just discussed).

**Pull request closed without merging**